### DISCUSSION TITLE
Window with error appear only for a second. Not sure it's bug or not

### DISCUSSION THREAD

**Discussion by Set27** [sveltejs/kit#12038] [Q&A]
When I run server for first time, window with error appears normally. But after reload, it's appear for a second.
I add ```server: {
    hmr: {
      overlay: true
    }
  }```
in vite.config.ts explicitly, but it does not help

Demo

https://github.com/sveltejs/kit/assets/49077110/d81c9c06-39eb-4a01-b98b-5c8ef47fbe41

**Comment by teemingc** [comment]
Can you provide a minimal reproduction code repository? It's difficult to diagnose the issue from a screen recording.

**Reply by Set27** [reply]
I can provide specific configuration, can't provide repo, sorry
